frontends/Jenkinsfile

```diff
@@ -24,73 +24,92 @@ pipeline {
         timestamps()
     }
     environment {
-        MASTER_BRANCH="master"
-        MASTER_STABLE_BRANCH="master-stable"
-
         NODE_BASE_IMAGE="registry.access.redhat.com/ubi9/nodejs-18:1-53"
         CYPRESS_TEST_IMAGE="quay.io/cloudservices/cypress-e2e-image:06b70f3"
 
         CICD_URL="https://raw.githubusercontent.com/RedHatInsights/cicd-tools/main"
+
+        IMAGE="${APP_IMAGE}"
+
     }
 
     stages {
         stage('Clone Git Repository') {
             agent { label 'insights' }
             steps {
                 script {
-                    sh '''
-                        env
-                    '''
+                    GIT_COMMIT_HASH = ghprbActualCommit.take(6)
```

Comment:
Please, as discussed, review if this is necessary and if it should be 6 chars instead of the usual 7

Reply:
I think there was a reason for this but in order for me to check, I'd like to do it in a follow up PR if possible.

Reply:
lets check this ASAP please